.github/workflows/file-size-impact/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "single-spa-workflow-file-size-impact",
+  "author": "Tiago Stutz",
+  "version": "0.0.1",
+  "description": "",
+  "main": "report-file-size-impact.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@jsenv/file-size-impact": "4.3.0"
```

Comment:
By the way latest version is now `5.3.0`. Despites the major version I see nothing in your code that needs to be updated if you upgrade to 5.3.